test/doom.d/init.el

```diff
@@ -104,6 +104,7 @@
        ;;taskrunner        ; taskrunner for all your projects
        ;;terraform         ; infrastructure as code
        ;;tmux              ; an API for interacting with tmux
+       tree-sitter       ; syntax and parsing, sitting in a tree...
```

Comment:
Enabling this module so I can ensure that it is building correctly in CI in the future.